intake_netcdf/__init__.py

```diff
@@ -33,18 +37,21 @@ class NetCDFSource(base.DataSource):
 
     Parameters
     ----------
-    param: type
-        description
+    urlpath: str
+        Path to source file.
+    chunks: int or dict
+        Chunks is used to load the new dataset into dask
+        arrays. ``chunks={}`` loads the dataset with dask using a single
+        chunk for all arrays.
     """
-    container = 'python'
 
-    def __init__(self, urlpath, xarray_kwargs=None, metadata=None):
+    def __init__(self, urlpath, chunks, xarray_kwargs=None, metadata=None):
         self.urlpath = urlpath
+        self.chunks = chunks
         self._kwargs = xarray_kwargs or {}
-        self.chunks = self._kwargs.get('chunks') 
         self._ds = None
         super(NetCDFSource, self).__init__(
-            container=self.container,
+            container=None,
```

Comment:
Not xarray?

Reply:
The container is not used since we are overloading the `read*` methods (Intake plugins the "hard" way).  I'd rather give no information rather than false information.